lib/stream_data.ex

```diff
@@ -296,6 +317,10 @@ defmodule StreamData do
 
   ## Generator modifiers
 
+  # TODO: We have functions about `size` but so far we haven't explained
+  # what `size` does and why it matters. Maybe we need to discuss them on
+  # the shrinking section in the module doc.
```

Comment:
We definitely need to discuss this, good point. Especially because contrary to what one intuitively thinks, size is not related to shrinking in any way (that is, we don't make size smaller in order to shrink). :)